week 03/open beers/beerdetails.php

```diff
@@ -0,0 +1,153 @@
+<?php
+	
+	if (isset($_GET)) {
+		$id = $_GET["id"];
+	} else {
+		header("Location: index.php");
+	}
+
+	require_once("connection.php");
+
+	function getCompanyInfo($conn, $id) {
+    	$sql = "SELECT * FROM breweries WHERE id = $id";
+    	$result = $conn->query($sql);
+
+    	if ($result->num_rows > 0) {
+	        while($row = $result->fetch_assoc()) {
+	            echo "<p>";
+	            	echo "<span>";
+	            		echo "Address: ";
+		            	echo $row["address1"];
+		            	if (!empty($row["address2"])) {
+		            		echo " / " . $row["address2"];
+		            	}
+		            	echo ", " . $row["city"] . " " . $row["state"] . " " . $row["code"] . ", " . $row["country"];
+	            	echo "</span>";
+	            	echo "<span>";
+	            		echo "Phone: ";
+		            	echo $row["phone"];
+	            	echo "</span>";
+	            	if (!empty($row["website"])) {
+	            		echo "<span>";
+		            		echo "Website: ";
+			            	echo $row["website"];
+		            	echo "</span>";
+	            	}
+	            echo "</p>";
+	            if (!empty($row["descript"])) {
+	            	echo "<h5>";
+			           	echo "Website: ";
+				        echo $row["descript"];
+		           	echo "</h5>";
+	            }
+	        }
+	    } else {
+	        echo "0 results";
+	    }
+    }
+
+    function getName($conn, $id) {
+    	$sql = "SELECT * FROM breweries WHERE id = $id";
+    	$result = $conn->query($sql);
+
+    	if ($result->num_rows > 0) {
+	        while($row = $result->fetch_assoc()) {
+	            return $row["name"];
+	        }
+	    } else {
+	        echo "0 results";
+	    }
+    }
+?><!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="content-type" content="text/html" charset="UTF-8">
+	<title>
+		<?php
+			echo "Details | " . getName($conn, $id);
+		?>
+	</title>
+	<link href="https://fonts.googleapis.com/css?family=IBM+Plex+Serif" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="https://template68191.motopreview.com/mt-includes/css/assets.min.css">
+	<link rel="stylesheet" type="text/css" href="https://template68191.motopreview.com/mt-demo/68100/68191/mt-content/assets/styles.css?_build=1550004938">
+	<link rel="stylesheet" type="text/css" href="css/details.css">
+</head>
+<body>
+	<div class="title">
+		<div class="inner">
+			<h1>
+				<?php
+					echo getName($conn, $id);
+				?>
+			</h1>
+			<?php
+				getCompanyInfo($conn, $id)
+			?>;
+		</div>
+	</div>
+
+	<div class="padding">
+	<?php
+		$sql = "SELECT *, breweries.name AS brewery,

```

Comment:
Probeer query-logica uit je templates te halen en zoveel mogelijk bovenaan je pagina te schrijven (of nog beter: op termijn in je klassen)